python/graphstorm/config/argument.py

```diff
@@ -1492,6 +1492,18 @@ def eval_metric(self):
 
         return eval_metric
 
+    @property
+    def num_gnn_ngnn_layers(self):
```

Comment:
why do you not call it `num_mlp_layers_in_gnn`? how many people know what ngnn is? also, does ngnn refer to the whole model architecture or the MLP?

Reply:
I think it is better to call it NGNN here. First, it would match the name with the work we did previously here: https://arxiv.org/pdf/2111.11638.pdf. Second, I think mlp layers in gnn would sound like we add new layers in the current GNN. But what we want to do is just adding the option to efficiently increase parameters. I suggest still using num_gnn_ngnn_layers.

Reply:
how is what we did different from adding MLP layers?

Reply:
What we add is MLP Layer definitely. The only thing is that we put them in input_layers, gnn_layers, and decoder.

Reply:
i want to understand the benefit of using `num_gnn_ngnn_layers` over `num_mlp_layers_in_gnn`. when users see ngnn, they will ask what ngnn is. they will need to check out the paper to figure out what this argument actually does. if we just call it `num_mlp_layers_in_gnn`, it clearly tells users that they are adding some MLP layers in the GNN layer. we can choose other names. my point is that having ngnn in the name causes unnecessary confusion.